doc/FormattingDecisionComments.md

```diff
@@ -2,7 +2,15 @@
 
 This is a document explaining our reasoning behind the formatting decision for comments.
 
-## Conventions in Documentation
+Currently `erlfmt` moves all trailing comments to the line above, so that all comments are always alone on their own line.
+
+After looking at documentation and doing an analyses we have concluded that [directly following comments](#single-comments-on-the-same-line) should also be supported in future.
+This does not include aligning comments, since:
+  - we could not find a popular enough convention of aligning to a specific column number and
+  - aligning comments can cause large diffs, from changing a single line for comments to be re-aligned.
+More details on this future change can be found in issue [#219](https://github.com/WhatsApp/erlfmt/issues/219)
+
+## Documentation
```

Comment:
You can also add [this guideline](https://github.com/inaka/erlang_guidelines#comment-levels) to the documentation examples in your list.

Reply:
Thank you, great idea :)